BUILD

```diff
@@ -623,7 +665,15 @@ grpc_cc_library(
         "src/cpp/common/tls_credentials_options.cc",
         "src/cpp/server/insecure_server_credentials.cc",
         "src/cpp/server/secure_server_credentials.cc",
-    ],
+    ] + select(
```

Comment:
This needs to be select_deps - our tooling for imports can't support selects here

Reply:
I see.